front/app/components/admin/ActionForm/FlowVisualization/Blocks/index.tsx

```diff
@@ -116,38 +118,33 @@ const Blocks = ({
               number={
                 2 + enabledSteps(emailConfirmationEnabled, verificationEnabled)
               }
-              text={formatMessage(messages.completeTheExtraQuestionsBelow)}
+              text={formatMessage(
+                messages.completeTheDemographicQuestionsAbove
+              )}
             />
           </>
         )}
       </>
     );
   }
 
-  // TODO: Fix this the next time the file is edited.
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-  if (permittedBy === 'verified') {
-    return (
-      <>
-        <SSOBlock
-          verificationExpiry={verificationExpiry}
-          onChangeVerificationExpiry={onChangeVerificationExpiry}
-        />
-        {showCustomFields && !userFieldsInForm && (
-          <>
-            <Edge />
-            <Block
-              number={2}
-              text={formatMessage(messages.completeTheExtraQuestionsBelow)}
-            />
-          </>
-        )}
-      </>
-    );
-  }
-
-  // Unreachable
-  return null;
+  return (
```

Comment:
Very minor but wonder if it might make more sense for `permitted_by === 'users'` to be the default returned `<SSOBlock>` here given that it is default in the interface